src/network/websocket/WebSocketProtocol.h

```diff
@@ -85,9 +88,38 @@ class WebSocketProtocol {
 
 	void addDisconnectionHandler(const connhandler_t& handler);
 
+	/**
+	 * @brief Set the handler that's called when the heartbeat times out.
+	 *
+	 * If the heartbeat is reestablished after timing out, and then times out again, this
+	 * handler will be called again.
+	 *
+	 * @param timeout The heartbeat timeout.
```

Comment:
Better to write something along the lines of the "The time in milliseconds after which the heartbeat times out"